addons/website_project_issue/views/project_issue_templates.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="portal_project_issue" name="Portal Project Issues" inherit_id="website_portal.account">
+        <xpath expr="//div[@id='documents_list']/*[last()]" position="after">
+            <h3 class="page-header">Your Issues</h3>
+            <t t-if="not issues">
+                <p>There are currently no issues for your account.</p>
+            </t>
+            <t t-if="issues">
+                <table class="table table-hover status_table">
+                    <thead>
+                        <tr class="active">
+                            <th>Issue #</th>
+                            <th>Description</th><th/>
+                            <th>Assigned To</th>
+                        </tr>
+                    </thead>
+                    <t t-foreach="issues" t-as="issue">
+                        <tr t-att-class="'hidden to_hide' if issue_index &gt; 4 else ''">
```

Comment:
@rim-odoo ,

we implemented same like : https://github.com/odoo/odoo/blob/master/addons/website_portal/views/templates.xml#L66

also I agree we should fix limit from controller side but right now we do not have any JS widget to load more records. so we implemented same like website_portal module.